src/pages/Components/Notification/code.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 'Code'
+order: 2
+---
+
+## Import
+
+```css
+@import 'settings-tools/_all-settings';
+@import 'components/_c.notification';
+```
+
+## Basic usage
+
+Wrap a `h3`, `p` and a `a` or `button` tags in a `div` and apply the `mc-notification` class:
```

Comment:
still H3 here, the code review was not done previously.

Reply:
The H level will be determined by the context and SEO requirements, and not set as the same in all notifications.

Reply:
I'm gonna try to explain this in the doc, but surely the doc will be subject of further discussions.